src/services/rewardHistory.ts

```diff
@@ -0,0 +1,52 @@
+import config from "config";
+import axios from "axios";
+import { Pool } from "pg";
+import { Request, Response } from "express";
+
+import { Dictionary } from "../utils"
+
+const addressesRequestLimit:number = config.get("server.addressRequestLimit");
+
+interface RewardTple {
+    epoch: number;
+    amount: number;
+}
+
+
+const rewardAddrQuery = `
```

Comment:
This is only part of the problem.
The reward history should be the sum of MIR certificates taken from the reserved along AND the reward distributed by the protocol (these are implicit and not part of any transaction)
Probably this is related: https://github.com/input-output-hk/cardano-db-sync/issues/198